pcdsdevices/sqr1.py

```diff
@@ -0,0 +1,43 @@
+"""
+Ophyd class for square-one tri-sphere motion system (https://www.sqr-1.com/tsp.html)
+"""
+
+import logging
+
+from ophyd.device import Device
+from ophyd.device import FormattedComponent as FCpt
+from ophyd.pv_positioner import PVPositionerIsClose
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+logger = logging.getLogger(__name__)
+
+
+class SQR1Axis(PVPositionerIsClose):
+    setpoint = FCpt(EpicsSignal, '{self.prefix}:TARGET:{self._axis}', kind='normal')
+    readback = FCpt(EpicsSignal, '{self.prefix}:TARGET:{self._axis}:RBV', kind='normal')
+    actuate = FCpt(EpicsSignal, '{self.prefix}:MOV', kind='normal')
+    actuate_value = 1
+    stop_signal = FCpt(EpicsSignal, '{self.prefix}:KILL', kind='normal')
+    stop_value = 1
+
+    llm = FCpt(EpicsSignalRO, '{self.prefix}:TARGET:{self._axis}:LLM', kind='normal')
+    hlm = FCpt(EpicsSignalRO, '{self.prefix}:TARGET:{self._axis}:HLM', kind='normal')
+
+    def __init__(self, axis, **kwargs):
+        self._axis = axis  # axis values {X,Y,Z,rX, rY, rZ}
+        super().__init__(**kwargs)
+
+        self.setpoint.put(self.readback.get())  # sync position setpoint with readback
+        self._limits = ((self.llm.get(), self.hlm.get()))
```

Comment:
Same as above: does this need to happen now, during `__init__`? This is a 2x timeout + 6 axis = 12 timeouts, potentially. It might be better to do this when the limits are checked or by using a subscription callback.